### PR TITLE
Add modernhealth [1757773]

### PR DESCRIPTION
**Pull request by dividehex**
~Before merging this, I need to confirm `authorized_groups` is set to the correct scope.~

Group scope is correct.

### CODE REVIEW

**Comment by dividehex** [comment]
Also needs an image

File: apps.yml [hunk 1]
```diff
@@ -4018,3 +4018,15 @@ apps:
     url: https://auth.mozilla.auth0.com/samlp/FoeZGnqq0X9fQXQQ9Ve5kdSdPN4eedlG
     vanity_url:
     - /prefect-pocket
+- application:
+    authorized_groups:
+    - team_moco_benefited
```
Comment (floatingatoll) [inline apps.yml:4023]: Does mofo expect to have access to this as 'benefited employees' in general? If so, team_mofo, I believe, is all that's required (they don't differentiate benefited beyond that).

**Review by floatingatoll — APPROVED** [review]
r+ with caveat noted about mofo

**Comment by floatingatoll** [comment]
Consider fixing the typo in [de0c2fb](https://github.com/mozilla-iam/sso-dashboard-configuration/pull/549/commits/de0c2fba13d0cdf21ea3ef011e9d445b2c6fa558) description as well, though not strictly necessary.

File: apps.yml (hunk 1 above)
Comment (dividehex) [reply apps.yml:4023]: I'll double check on whether mofo should be included

Comment (dividehex) [reply apps.yml:4023]: MoFo will not be covered.  It’s a completely different legal entity and they offer their own set of benefits.

**Comment by floatingatoll** [comment]
Replaced by #559.

**Pull request closed without merging**